content/post/data-warehouse-sla-p1.md

```diff
@@ -0,0 +1,64 @@
+---
+author: Scott Breitenother
+title: "Should Your Data Warehouse Have an SLA? (Part 1)"
+cover: "/img/data-warehouse-sla-p1.jpg"
+publishDate: "2018-05-28"
+draft: true
+tags: 
+ - analytics
+ - data warehouse
+ - SLA
+---
+
+Yes, if you want to build a truly data-driven organization your data warehouse needs a Service Level Agreement (SLA). At the core of any data driven organization is trust - your stakeholders must trust that when they need data, it will be there and it will be accurate. Without trust in the data warehouse, your organization will be less likely to use data to drive decisions big and small.
+
+In my previous post [Reporting is a Gateway Drug]({{< relref "reporting-is-a-gateway-drug.md" >}}) I explored reporting as a tool to build a trusting stakeholder relationship. In this post I explore trust through the concept of a data warehouse SLA. In part two I explore the people, process and tools you need to successfully implement the SLA.
+<!--more-->
+
```

Comment:
It might help to start with a motivating counter-example. E.g., what does the world look like without an SLA? That might then help to set-up and tie the following sections together.